src/pages/TipsEdit/TipsEdit.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+//style
+import "./TipsEdit.sass";
+//components
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+export const TipsEdit = ({}) => {
+  return (
+    <div className="container">
+      <div />
+
+      <form noValidate autoComplete="off">
+        <TextField className="title" id="outlined-basic" label="Title" variant="outlined" />
```

Comment:
Con Material Ui, dentro de un form, elementos de formulario deberían ir dentro de un [Form Control](https://material-ui.com/api/form-control/)
 
```suggestion
<FormControl>
  <InputLabel htmlFor="my-input">Email address</InputLabel>
  <Input id="my-input" aria-describedby="my-helper-text" />
  <FormHelperText id="my-helper-text">We'll never share your email.</FormHelperText>
</FormControl>
```

Lo anterior es un ejemplo. Se usa un `FormControl` por cada elemento de formulario.

Por otro lado, el `TextField` es el equivalente al `text-area`, si tenemos que ingresar una sola línea tenemos que usar `Input`